src/goto-analyzer/unreachable_instructions.cpp

```diff
@@ -90,7 +90,7 @@ static void add_to_json(
 
   entry["function"]=json_stringt(id2string(end_function->function));
   entry["fileName"]=
-    json_stringt(concat_dir_file(
+    json_stringt(fileutil_concat_dir_file(
```

Comment:
I'm routinely allergic to intrusive changes: would it be possible to skip the renaming? If ambiguity really turns out to be a problem, a class (with static members) should be introduced.